package ExceptionHandiling;

public class Test3 {

	public static void main(String[] args) {
		try {
		int i=1/0;
	//	}
	//	catch(Exception a) {
	//		System.out.println("handled");
		}
    	finally {
    		System.out.println("Hello");
    		System.out.println("bye");
    	}

	}

}
